Randolph-dev/Druid_Partnering_Project_React24k
# Add paragraph content for jobspage

## Related ticket:
[#139](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-139?atlOrigin=eyJpIjoiYjFhZDc2ODBkNGI4NDVjZDg2M2ViMjU5OTllZjNjNjUiLCJwIjoiaiJ9)

In this PR:
Content for Jobspage

## Testing instruction
- Checkout branch
- Backend: Import db and config
- Check the front-page
